README.md

```diff
@@ -614,13 +615,14 @@ This output writes data to the [Elasticsearch](https://www.elastic.co/products/e
 | `type` | "ElasticSearch" | Yes | Specifies the output type. For this output, it must be "ElasticSearch". |
 | `indexNamePrefix` | string | No | Specifies the prefix to be used when creating the Elasticsearch index. This prefix, together with the date of when the data was generated, will be used to form the name of the Elasticsearch index. If not specified, a prefix will not be used. |
 | `serviceUri` | URL:port | Yes | Specifies where the Elasticsearch cluster is. This is needed for EventFlow to locate the cluster and send the data. Single URL or semicolon-separated URLs for connection pooling are accepted |
+| `connectionPoolType` | "SingleNode", "Static", "Sniffing", or "Sticky" | Yes | Specifies the Connection Pool that takes care of registering what nodes there are in the cluster. |
```

Comment:
1. How about we do not support SingleNode at all? I think if you just use one URI with a Static connection pool, that is effectively the same as a SingleNode pool
2. Let's default to Static connection pool if it is not specified otherwise in the configuration

Reply:
Looking through the elastic documentation does not really give you an answer. Looking at the code does though. 

A SingleNode connection does not allow you to use the ElasticClient.Ping(). 

Since the ElasticClient is not exposed to users, not being able to ping does not seem like an issue. So in short, I agree. I will remove the singleNode option.

On point 2, if we make "Static" the default, should we make this a optional configuration value? I would think yes, to remove the need for this elastic domain  knowledge.

Reply:
Yes, let's make it optional